scalatest/src/main/scala/org/scalatest/tools/Memento.scala

```diff
@@ -136,7 +136,7 @@ private[tools] object Memento {
   // Constructs a Memento object from a single-line string.
   //
   def fromString(str: String): Memento = {
-    val splits = str.split(" ")
+    val splits = str.split(" ").map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.